components/email/test/editorial_signup.coffee

```diff
@@ -63,18 +87,18 @@ describe 'EditorialSignupView', ->
         ARTSY_EDITORIAL_CHANNEL: '123'
         SUBSCRIBED_TO_EDITORIAL: false
       @view.initialize()
-      _.defer ->
-        @view.showEditorialCTA.called.should.not.be.ok()
```

Comment:
Looks like these weren't really testing anything. The defer didn't matter because there wasn't anything waiting in the callstack and showEditorialCTA would have failed without because we haven't stubbed it